### PR TITLE
github: added .jira_sync_config.yaml for the synchronization with Jira

### PR DESCRIPTION
**Pull request by mseralessandri**
`.jira_sync_config.yaml` is a configuration file to allow the creation of Jira issues when a particular tag (`Jira`) is assigned to the GitHub issue. It uses [gh-jira-sync-bot](https://github.com/canonical/gh-jira-sync-bot). The issues in Jira are grouped in a specific epic defined in the config file, but the parent epic can be changed without affecting the synchronization. The default issue type assigned in Jira is "Bug" but it can be changed assigning a different label (`feature`,`improvement`, `investigation`) to the GitHub issue.

### CODE REVIEW

**Review by roosterfish — COMMENTED** [review]
Looks good to me.

However I am wondering who is hosting the bot https://github.com/canonical/gh-jira-sync-bot? Is it setup generally for all projects under `canonical/*`?

**Review by roosterfish — APPROVED** [review]

**Comment by roosterfish** [comment]
@mseralessandri the repo requires your commits to be signed. You can use your already existing SSH key, see https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key#telling-git-about-your-ssh-key

**Comment by mseralessandri** [comment]
> @mseralessandri the repo requires your commits to be signed. You can use your already existing SSH key, see https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key#telling-git-about-your-ssh-key

The commit was signed, could you check again? Thanks!

**Comment by roosterfish** [comment]
> The commit was signed, could you check again? Thanks!

Yes your commit has the sign-off message, but the commit itself has to be signed with any of the methods in https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits. 

I find it the easiest to just use your existing SSH key as described in https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key#telling-git-about-your-ssh-key. 
Afterwards you can run `git commit -S --amend` which will sign your commit so you can `git push -f` to force push the commit on GitHub. 

GH is blocking the merge until all the commits of the PR are signed. See the red error message below.

**Review by roosterfish — APPROVED** [review]

**Pull request merged**
